src/routes/quotes/data.json

```diff
@@ -1,25 +1,139 @@
+    "author": "Vladimir Putin",
+    "author_description": "President of Russia",
+    "color": "white"
+  },
+  {
+    "background": "qiang",
+    "text":"AI must be guided in a direction that is conducive to the progress of humanity. So there should be a red line in AI development, a red line that must not be crossed. [...] It should not just benefit only a small group of people, but benefit the overwhelming majority of mankind. [...] It is essential that we work together and coordinate with each other.",
+    "author": "Li Qiang",
+    "author_description": "China's Head of Government",
+    "color": "black"
+  },
+  {
+    "background": "leyen",
+    "text": "[We] should not underestimate the real threats coming from AI. [Fully quoted the above statement on the risk of extinction.] [...] It is moving faster than even its developers anticipated. [...] We have a narrowing window of opportunity to guide this technology responsibly.",
+    "author": "Ursula von der Leyen",
+    "author_description": "Head of the executive branch of the European Union",
+    "color": "white"
+  },
+  {
+    "background": "guterres",
+    "text": "AI poses a long-term global risk. Even its own designers have no idea where their breakthrough may lead. I urge [the UN Security Council] to approach this technology with a sense of urgency. Unforeseen consequences of some AI-enabled systems could create security risks by accident. Generative AI has enormous potential for good and evil at scale. Its creators themselves have warned that much bigger, potentially catastrophic and existential risks lie ahead. Without action to address these risks, we are derelict in our responsibilities to present and future generations.",
+    "author:": "António Guterres",
+    "author_description": "Chief Executive Officer of the United Nations",
+    "color": "black",
+    "padding": "0rem"
+  },
+  {
+    "background": "pope",
+    "text": "I urge the global community of nations to work together in order to adopt a binding international treaty that regulates the development and use of artificial intelligence in its many forms. [...] [AI] may pose a risk to our survival and endanger our common home",
+    "author": "Pope Francis",
+    "author_description": "Head of the Catholic Church",
+    "color": "white",
+    "padding": "0rem"
+  },
+  {
+    "background": "kamala",
+    "text": "[...] just as AI has the potential to do profound good, it also has the potential to cause profound harm. From AI-enabled cyberattacks at a scale beyond anything we have seen before to AI-formulated bio-weapons that could endanger the lives of millions, these threats are often referred to as the “existential threats of AI” because, of course, they could endanger the very existence of humanity. These threats, without question, are profound, and they demand global action.",
```

Comment:
These two might be better with white?

<img width="332" alt="image" src="https://github.com/joepio/pauseai/assets/61096713/1dbe4691-3704-403c-a472-fc89ce98d21f">